realm/realm-library/src/main/cpp/io_realm_internal_Table.cpp

```diff
@@ -609,6 +609,21 @@ JNIEXPORT void JNICALL Java_io_realm_internal_Table_nativeSetObjectId(JNIEnv* en
     CATCH_STD()
 }
 
+JNIEXPORT void JNICALL Java_io_realm_internal_Table_nativeSetUUID(JNIEnv* env, jclass, jlong nativeTableRefPtr,
+                                                                      jlong columnKey, jlong rowKey, jstring j_value,
+                                                                      jboolean isDefault)
+{
+    TableRef table = TBL_REF(nativeTableRefPtr);
+    if (!TYPE_VALID(env, table, columnKey, type_UUID)) {
```

Comment:
Is it sane to silently ignore operations on invalid type?

Reply:
I don't think that is a good practice neither. I am following the same pattern as the other data types to get the same behavior. 

We should address this on a different PR.